sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12.integration.service.ts

```diff
@@ -250,6 +259,17 @@ export class IER12IntegrationService extends SFTPIntegrationBase<void> {
       ierFileDetail.applicationEventDate = ierRecord.applicationEventDate;
       ierFileDetail.currentOfferingId = ierRecord.currentOfferingId;
       ierFileDetail.parentOfferingId = ierRecord.parentOfferingId;
+      ierFileDetail.returnTransportationCosts =
+        ierRecord.returnTransportationCosts
+          ? combineDecimalPlaces(ierRecord.returnTransportationCosts)
+          : 0;
+      ierFileDetail.extraLocalTransportationCosts =
+        ierRecord.extraLocalTransportationCosts
+          ? combineDecimalPlaces(ierRecord.extraLocalTransportationCosts)
+          : 0;
+      ierFileDetail.extraShelterCosts = ierRecord.extraShelterCosts
+        ? combineDecimalPlaces(ierRecord.extraShelterCosts)
+        : 0;
```

Comment:
This logic is repeated many times. Please add a small method to avoid the repetition.